string baseLat
string baseLon
string roverLat
string roverLon
string destLat
string destLon
